runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsConfig.java

```diff
@@ -52,4 +52,14 @@ default boolean autopause()
 	{
 		return false;
 	}
+	@ConfigItem(
```

Comment:
Config items are always separated by a blank line.
```suggestion

	@ConfigItem(
```